### PR TITLE
EASY-1277: directory pointers with '.' fixed and introduce better-files

### PR DESCRIPTION
**Pull request by rvanheest**
fixes EASY-1277

This PR will replace #107: the problem we try to solve there is already solved by the better-files library.

#### When applied it will
1. process `Paths.get(".")` correctly (as described in the issue)
2. make `./clear-run.sh` run quietly
3. introduce better-files to solve (1)

* [x] await PR #106 to be finished to avoid merge conflicts with this one

@DANS-KNAW/easy for review

### CODE REVIEW

File: clear-run.sh [hunk 1]
```diff
@@ -18,6 +18,7 @@
 LOCAL_ARGS=$@
 OUTPUT=data/output
 STAGING=data/staging
+QUIET="-q"
```
[the remark this answers is not in the document]
Comment (janvanmansum) [reply clear-run.sh]: @rvanheest : I think Codacy is right here. This variable will probably not be visible in `run.sh` unless you export it.

**Review by janvanmansum — CHANGES_REQUESTED** [review]
@rvanheest Have you tested the `QUIET` variable? See comment by Codacy. For the rest I am sure this PR is fine.

**Comment by rvanheest** [comment]
I copied this from your code:

https://github.com/DANS-KNAW/dans-dev-tools/blob/master/run-quiet.sh

If you want me to do it differently, let me know!

**Comment by janvanmansum** [comment]
@rvanheest : the difference is that in https://github.com/DANS-KNAW/dans-dev-tools/blob/master/run-quiet.sh the variable is defined in the command line that calls the script, which makes it part of the environment that the scripts runs in. 

To test this:
```bash
#/bin/sh
#script1.sh

echo "separate line, not passed to script2"
MYVAR=1
./script2.sh

echo "same line, part of script2"
MYVAR=2 ./script2.sh

echo "export, visible to script2"
export MYVAR
./script2.sh
```

```bash
#!/bin/sh
#script2.sh
echo $MYVAR
```

**Comment by janvanmansum** [comment]
@rvanheest : Travis runs into the same problem that I e-mailed you about.

**Comment by rvanheest** [comment]
@janvanmansum regarding `QUIET`, I tested both versions (with and without `export`) and both of them work the same way... I changed it to add the `export` keyword, though, to make both you and @codacy-bot happy.

**Pull request merged**